rules/rules-overridden-azure/cloud-readiness/tests/socket-communication-target-azure-appservice.windup.test.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<ruletest id="socket-communication-test" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Could your format the XML headers? Something like:
<ruletest id="socket-communication-test" 
       xmlns="http://windup.jboss.org/schema/jboss-ruleset"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">

Reply:
Updated as suggested